refinery/core/tests.py

```diff
@@ -103,6 +102,36 @@ def setUp(self):
             self.username2, '', self.password2
         )
         self.get_credentials()
+
+    def get_credentials(self):
+        """Authenticate as self.user"""
+        # workaround required to use SessionAuthentication
+        # http://javaguirre.net/2013/01/29/using-session-authentication-tastypie-tests/
+        return self.api_client.client.login(
+            username=self.username,
+            password=self.password
+        )
+
+
+class ApiResourceTest(LoginResourceTestCase):
+
+    def setUp(self):
```

Comment:
This super call might be redundant, but still potentially good to have if someone ever extends the `setUp` here.

Reply:
I guess I was interested in just doing the same thing the other tests here were doing, so I'll keep it this way, but thanks!